actions/templates/partials/fonts.html

```diff
@@ -19,7 +19,8 @@
 {# [3] #}
 <link rel="stylesheet"
       href="https://fonts.googleapis.com/css2?family=Public+Sans:wght@400;500;600;700&display=swap"
-      media="print" onload="this.media='all'" />
+      {# media="print" onload="this.media='all'" #}
```

Comment:
Should that bit be commented out?

Reply:
Yes, I need to replace the Google Fonts with local fonts, but until that work is done, this inline scripting should be removed.

Linked issue: https://github.com/opensafely-core/actions-registry/issues/127